.github/workflows/publish-nix-pgupgrade-bin-flake-version.yml

```diff
@@ -113,7 +113,7 @@ jobs:
   
       - name: Upload pg_upgrade scripts to s3 prod
         run: |
-          aws s3 cp /tmp/pg_upgrade_bin.tar.gz s3://${{ secrets.PROD_ARTIFACTS_BUCKET }}/upgrades/postgres/supabase-postgres-${{ steps.process_release_version.outputs.version }}/20.04.tar.gz
+          aws s3 cp /tmp/pg_upgrade_bin.tar.gz s3://${{ secrets.PROD_ARTIFACTS_BUCKET }}/upgrades/postgres/supabase-postgres-${{ steps.process_release_version.outputs.version }}/24.04.tar.gz
```

Comment:
Ditto. Deduplication is needed.

Reply:
yes, agree we should address this. As stated above, as @pcnc had discussed in slack thread, these are currently tied to infrastructure functionality. So, instead of trying to make those changes as a part of the upgrade to ubuntu 24.04 PR, let's create another PR where we can introduce those changes and test and make sure nothing else breaks. We should not try to accomplish that in this PR.